experiment/build/gcb_build.py

```diff
@@ -39,84 +42,95 @@
 logger = logs.Logger('builder')  # pylint: disable=invalid-name
 
 
+def _get_buildable_images(fuzzer='', benchmark=''):
```

Comment:
Why default to empty string, what happens if the string is empty?

Reply:
The base images don't have any substitutions for fuzzer and benchmark, so I default that to empty to get base images.

Reply:
Empty string feels odd. Can you `None` instead?

Reply:
Noted.